From: Edward Cree <ecree@solarflare.com>
Date: Fri, 26 Aug 2016 15:12:57 +0100
Subject: sfc: remove duplicate assignment
Patch-mainline: v4.9-rc1
Git-commit: df0562c386c778c802b981fc4015a636aa5db866
References: bsc#1017967 FATE#321663

nic_data was already initialised to the right thing, no need to assign
it again.

Signed-off-by: Edward Cree <ecree@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/sfc/ef10.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/sfc/ef10.c
+++ b/drivers/net/ethernet/sfc/ef10.c
@@ -1727,7 +1727,6 @@ static int efx_ef10_try_update_nic_stats
 	efx_ef10_get_stat_mask(efx, mask);
 
 	dma_stats = efx->stats_buffer.addr;
-	nic_data = efx->nic_data;
 
 	generation_end = dma_stats[MC_CMD_MAC_GENERATION_END];
 	if (generation_end == EFX_MC_STATS_GENERATION_INVALID)
